### PR TITLE
Alpha fixup

### PR DESCRIPTION
**Pull request by soreau**

### CODE REVIEW

File: plugins/single_plugins/alpha.cpp [hunk 1]
```diff
@@ -99,6 +77,47 @@ class wayfire_alpha : public wayfire_plugin_t
...
+
+        if (ev->orientation == WLR_AXIS_ORIENTATION_VERTICAL)
+            update_alpha(view, ev->delta);
+    };
+
+    wf_option_callback min_value_changed = [=] ()
+    {
+        output->workspace->for_each_view([=] (wayfire_view view)
+        {
+            wf_2D_view *transformer;
```
Comment (ammen99) [inline plugins/single_plugins/alpha.cpp]: why not move that below (`wf_2D_view *transformer = ...`)

File: plugins/single_plugins/alpha.cpp [hunk 2]
```diff
@@ -99,6 +77,47 @@ class wayfire_alpha : public wayfire_plugin_t
...
+
+            if (!view->get_transformer("alpha"))
+                return;
+
+            transformer = dynamic_cast<wf_2D_view*> (view->get_transformer("alpha").get());
+
+            if (transformer->alpha < min_value->as_double())
+                transformer->alpha = min_value->as_double();
+
+            view->damage();
```
Comment (ammen99) [inline plugins/single_plugins/alpha.cpp]: theoretically no need to damage if the alpha wasn't changed, could be moved inside the `if` on line 113

File: plugins/single_plugins/alpha.cpp (hunk 1 above)
Comment (soreau) [reply plugins/single_plugins/alpha.cpp]: I was trying to keep declarations before code. Changed to the way you suggest.

File: plugins/single_plugins/alpha.cpp (hunk 2 above)
Comment (soreau) [reply plugins/single_plugins/alpha.cpp]: True, fixed.

File: plugins/single_plugins/alpha.cpp (hunk 1 above)
Comment (ammen99) [reply plugins/single_plugins/alpha.cpp]: as a general notice this isn't done in C++ at all AFAIK, and certainly isn't done in wayfire except perhaps at a limited number of places.

Comment (soreau) [reply plugins/single_plugins/alpha.cpp]: Noted.

**Comment by ammen99** [comment]
Thanks!

**Pull request merged**